src/aiida/manage/configuration/settings.py

```diff
@@ -99,10 +116,13 @@ def get_configuration_directory_from_envvar() -> pathlib.Path | None:
     """
     environment_variable = os.environ.get(DEFAULT_AIIDA_PATH_VARIABLE)
 
+    default_dirpath_config = pathlib.Path(DEFAULT_AIIDA_PATH).expanduser() / DEFAULT_CONFIG_DIR_NAME
+
     if environment_variable is None:
-        return None
+        return default_dirpath_config
```

Comment:
I wouldn't change this function to return the default. I intentionally didn't do that because it may be useful to find out whether the env vars actually define a path or not. You can simply keep setting the default in the code that calls this function.